css/index.css

```diff
@@ -0,0 +1,184 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto";
+  font-family: 'Source Sans Pro';
+}
+
+
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.text-decoration-none {
+  text-decoration: none;
+}
+
+.btn {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+  border-radius: 68px;
+  padding: 10px;
+  text-decoration: none;
+  width: 100%;
+}
+
+.btn--primary {
+  background-color: #EF2A82;
+  color: #FFFFFF;
+}
+
+.btn--outline {
+  background-color: transparent;
+  border: 1px solid #8794BA;
+  color: #8794BA;
+}
+
+.c-secondary {
+  color: #8794BA;
+}
+
+.c-white {
+  color: #FFFFFF;
+
+}
+
+header {
+  background-color: #172755;
+
+}
+
+.header__top-top {  
+  background: #0088F080;
+  padding-top: 48px;
+  padding-bottom: 22px;
+}
+
+
+.header__top {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header__bottom {
+ padding-top: 28px;
+ padding-bottom: 38px;
+}
+
+
+.nav__menu {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  list-style: none;
+  gap: 36px;
+}
+
+.menu__link {
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 18px;
+  color: #8794BA;
+}
+
+.header__login {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 40px;
+}
+
+.header__btn {
+  width: 160px;
+}
+
+.header__logo img {
```

Comment:
Ya te había mencionado esto anteriormente. Si no tienes un estilo para `.header__logo`, usalo mejor en la imagen
<img width="810" alt="image" src="https://github.com/user-attachments/assets/984a3ac0-e8c0-4733-ae7b-2207956a292f">
```suggestion
.header__logo {
```